profiles/Pranay202.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/68046838/172583862-aab04eeb-d77f-42b3-8678-b05a23daf57e.png",
+    "issueId": 1161,
+    "name": "Pranay Kharabe",
+    "username": "Pranay202"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

updated profiles/Pranayk.json